### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by abdo-012** [community/community#37319] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi
I have a question how do i can get These Badges

**Comment by pascalfarrel** [comment, marked as answer]
I'm just passing along old Information. There are several ways to get badges here

- https://github.com/Schweinepriester/github-profile-achievements

- https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile

Currently I own a Pro badge by registering myself as a student through the link bellow

https://education.github.com/discount_requests/pack_application

Hope this helps :)

**Reply by abdo-012** [reply]
Thank you so much

**Reply by habolanos** [reply]
Great solution, Thanks

**Comment by tawfik-s** [comment]
you can search for each badge and understand the steps to get it 
you can find all badges here
https://github.com/tawfik-s/github-profile-achievements 

please close the question and mark the answers

**Comment by abdo-012** [comment]
Thank you guys
